### DISCUSSION TITLE
Where are the official docker images located?

### DISCUSSION THREAD

**Discussion by olwe0002** [patroni/patroni#3208] [Q&A]
In this repo multiple Dockerfiles are contained, e.g. on [root](https://github.com/patroni/patroni/blob/v3.3.4/Dockerfile) with the remark that it's for development purposes only, and in [kubernetes](https://github.com/patroni/patroni/blob/v3.3.4/kubernetes/Dockerfile), both defaulting to postgres 16.
Additionally, i'm aware of the spilo repo with its [package registry](https://github.com/orgs/zalando/packages?repo_name=spilo), but seems like this is not updated any more to newer versions of patroni or postgres.
Are there any official images in any registry, including bundled postgres + patroni, which are still maintained? with production ready images?

**Comment by CyberDem0n** [comment]
Dockerfiles in the kubernetes folder are also for development purposes only.
In this repo we are focusing only on developing Patroni and nothing else.
